client/components/product-icon/test/test.jsx

```diff
@@ -0,0 +1,104 @@
+jest.mock( 'lib/abtest', () => ( {
```

Comment:
It feels odd to have those at the top of the file. I'd expect them to be in a `beforeEach` in the corresponding `describe` call.